### DISCUSSION TITLE
Create pull request button is misplaced

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#45529] [Pull Requests]
GitHub Mobile beta

1. open
https://github.com/dart-lang/sdk/compare/main...jsoref:spelling?expand=1

### Actual results
![Screenshot_20230127-080929.png](https://user-images.githubusercontent.com/2119212/215094901-cf777dfd-ea07-4027-bd4c-5dd109683ad9.png)

### expected results
The green button should be on the next line...

**Comment by stevepopovich** [comment]
Moving this to `Pull Request` as `Mobile` is for native mobile app feedback. Thanks!

**Reply by jsoref** [reply]
I think the confusing thing to me is that I clicked the link *from* GitHub Mobile.
